packages/fast-usdc-contract/src/exos/liquidity-pool.js

```diff
@@ -199,7 +183,7 @@ export const prepareLiquidityPoolKit = (zone, zcf, USDC, tools) => {
       repayer: {
         /**
          * @param {TransferPart} sourceTransfer
-         * @param {RepayAmountKWR} split
+         * @param {import('@agoric/fast-usdc/src/utils/fees.js').RepayAmountKWR} split
```

Comment:
is this on purpose? you don't want to use `@import`?

Reply:
this was expedient. it had been ambient and the autocomplete fix was this. thinking you might be asking me to use `@import` I've gone ahead and done that.